### ISSUE TITLE
Tag Cloud visualization

### ISSUE THREAD

**Issue by heathergreerklein** [DSpace/dspace-angular#790]
From release plan spreadsheet

Estimate from release plan: 24 hours
Expressing interest: none

No additional notes

**Comment by arvoConsultores** [comment]
We are interested in this feature.  Any aspects to take into account before starting (has the label "need discussion")

**Comment by tdonohue** [comment]
@arvoConsultores : Currently, we (the Committers) are very uncertain if this feature is useful.  We had recommended talking to DCAT about whether anyone feels a "tag cloud" visualization is still useful.  That's why this is flagged as  "low priority", as we haven't (yet) heard of anyone saying they want DSpace to have a "tag cloud" again.  For example, there are no comments on this ticket asking for the feature, and no one has reached out on mailing lists asking for it.

So, honestly, we had previously talked about dropping this feature altogether & not supporting it in DSpace 7 or later.

That said, if you have clients that need this feature, then that may be a sign that this is useful still.  But, it may be that the feature needs to be possible to disable.  If you are also uncertain about its usefulness, then I'd recommend finding something different to work on.
